### PR TITLE
Set price of agreement when sourcing with a framework

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by lepistone** [comment]
@yvaucher this needs review or is it WIP? Can you label it please?

**Comment by yvaucher** [comment]
@lepistone as travis is failing it is WIP

**Comment by gurneyalex** [comment]
rebased

**Comment by gurneyalex** [comment]
with the tests fixed, :+1:

File: framework_agreement_sourcing/tests/test_agreement_souce_line_to_po.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ def setUp(self):
                 lid = self.requisition_line_model._generate_source_line(
                     cr, uid, line)
                 source_ids += lid
-        self.assertTrue(len(source_ids) == 2)
+        self.assertEqual(len(source_ids), 2,
```
Comment (lepistone) [inline framework_agreement_sourcing/tests/test_agreement_souce_line_to_po.py]: I don't think we really need to specify failure messages with `self.assert*` assertions.

Here `assertEqual` will automatically say, for example, that `len(source_ids)` is `0 != 2`, which is a good failure message.

**Comment by lepistone** [comment]
:+1: with a minor comment on assertions.
@yvaucher ping me for a merge, with or without a change in assertions. Thanks!

**Comment by yvaucher** [comment]
@lepistone fixed

**Pull request merged**